packages/blockchain-extension/extension/commands/addEnvironmentCommand.ts

```
@@ -62,6 +61,9 @@ export async function addEnvironment(): Promise<void> {
         }
         Reporter.instance().sendTelemetryEvent('addEnvironmentCommand');
     } catch (error) {
+        if (fabricEnvironmentEntry.name) {
```

You don't need this bit as the delete will ignore the error if it doesn't exist